include/poncos/scheduler_multi_app.hpp

```diff
@@ -1,15 +1,26 @@
 #ifndef scheduler_multi_hpp
 #define scheduler_multi_hpp
 
+#include <numeric>
```

Comment:
Please move the include to the cpp file ... unless it is required in the header.